test/test_operations.py

```diff
@@ -2642,14 +2642,14 @@ def test_api(self):
 
     ctx = torch_xla._XLAC.lowering.LoweringContext("MyCustomName")
     ctx.build([result])
-    hlo = ctx.hlo()
     hlo_text = ctx.hlo_text()
     self.assertIn('MyCustomName', hlo_text)
-    self.assertIn('opcode: "parameter"', hlo_text)
-    self.assertIn('opcode: "parameter"', hlo_text)
+    self.assertTrue(hlo_text.count('opcode: "parameter"'), 2)
     self.assertIn('opcode: "add"', hlo_text)
     mapping = ctx.parameter_id_tensor_mapping()
-    self.assertEqual(len(mapping), 2)
+    num_params = len(mapping)
+    self.assertEqual(num_params, 2)
+    self.assertTrue(met.metric_data("TransferFromDeviceTime"))
```

Comment:
Should we clear `met` before checking metric data/running this test case?

Reply:
Alternatively, check `TransferFromDeviceTime` is zero before `parameter_id_tensor_mapping`, and then checking it's larger after `parameter_id_tensor_mapping`.

Reply:
We have it in https://github.com/pytorch/xla/pull/8453/files#diff-b50b5ec9105ce166bf891d5d50080c03fef30396e6b63355ed69ae009064d3d6R22 already.

Reply:
I added to the older file, which you meant.